pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -139,6 +139,9 @@ func (c *cephStatusChecker) updateCephStatus(status *cephclient.CephStatus, cond
 	// Update with Ceph Status
 	cephCluster.Status.CephStatus = toCustomResourceStatus(cephCluster.Status, status)
 	cephCluster.Status.Phase = condition
+	cephCluster.Status.CephStatus.PgMap.TotalBytes = status.PgMap.TotalBytes
+	cephCluster.Status.CephStatus.PgMap.UsedBytes = status.PgMap.UsedBytes
+	cephCluster.Status.CephStatus.PgMap.AvailableBytes = status.PgMap.AvailableBytes
```

Comment:
when there is a failure condition (line[ 114](https://github.com/rook/rook/blob/4c36f79cf0e12ea86ef41f6444ef8ac0229b1c60/pkg/operator/ceph/cluster/cephstatus.go#L114) ) these values will be updated as 0.  Is that expected? Or should we skip adding usage when there is a failure condition?

Reply:
Per discussion, let's not change to 0 on failure. Instead, let's keep the previously set value and also have a "lastUpdated" timestamp with the capacity section.

Reply:
Done. Now, added check for status and then adding values.